packages/headless/src/app/engine-configuration.ts

```diff
@@ -110,11 +110,15 @@ export interface AnalyticsConfiguration {
    */
   originContext?: string;
   /**
-   * Origin level 2 is a usage analytics event metadata whose value should typically be the name/identifier of the tab from which the usage analytics event originates.
+   * Sets the value of the Origin Level 2 dimension on the analytic events.
    *
-   * When logging a Search usage analytics event, originLevel2 should always be set to the same value as the corresponding tab (parameter) Search API query parameter so Coveo Machine Learning models function properly, and usage analytics reports and dashboards are coherent.
+   * Origin level 2 is a usage analytics event metadata whose value should typically be the name/identifier of the tab from which the usage analytics event originates. 
    *
-   * This value is optional, and will automatically try to resolve itself from the tab search parameter.
+   * In the context of product listing, the value should match the breadcrumb of the product listing page from which the usage analytics event originates (for example, `canoes-kayaks/kayaks/sea-kayaks`).
+   *
+   * When logging a usage analytics event, originLevel2 should always be set to the same value as the corresponding `tab` (parameter) Search API query parameter so Coveo Machine Learning models function properly, and usage analytics reports and dashboards are coherent.
```

Comment:
Just so I follow: this means that even in the context of product listing, originLevel2 will match the `tab` Search API query param, which is also the breadcrumb value?

Reply:
@jpmarceau Yes!

Here's the payload of a listing query. You can see that the tab parameter also has the listing's breadcrumb as a value:
![Screenshot 2024-02-02 at 9 40 43 AM](https://github.com/coveo/ui-kit/assets/75028303/e7c084ad-5ae3-4461-8729-7fb598c476ac)